src/Modules/Reports/Commands.Reports/GetPowerBITile.cs

```diff
@@ -0,0 +1,102 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Abstractions.Interfaces;
+using Microsoft.PowerBI.Common.Api.Reports;
+using Microsoft.PowerBI.Common.Client;
+
+namespace Microsoft.PowerBI.Commands.Reports
+{
+    [Cmdlet(CmdletVerb, CmdletName, DefaultParameterSetName = ListParameterSetName)]
+    [OutputType(typeof(IEnumerable<Tile>))]
+    public class GetPowerBITile : PowerBIClientCmdlet, IUserScope, IUserId, IUserFirstSkip
+    {
+        public const string CmdletVerb = VerbsCommon.Get;
+        public const string CmdletName = "PowerBITile";
```

Comment:
Consider adding an alias for Get-PowerBIDashboardTile or renaming this to match your roadmap.